Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableEvent.cs

```diff
@@ -79,10 +79,17 @@ public Type ReceiverType
         private List<InspectorPropertySetting> Settings = new List<InspectorPropertySetting>();
 
         /// <summary>
-        /// Create the event and setup the values from the inspector
+        /// Create the event and setup the values from the inspector. If the asset is invalid,
+        /// returns null.
         /// </summary>
         public static ReceiverBase CreateReceiver(InteractableEvent iEvent)
         {
+            if (string.IsNullOrEmpty(iEvent.ClassName))
```

Comment:
Do you think we should check `iEvent` for null as well?

Reply:
I think IsNullOrEmpty covers that

Reply:
Won't `iEvent.ClassName` null ref before that?